cumulusci/cli/cci.py

```diff
@@ -854,11 +854,58 @@ def service_info(runtime, service_name, plain):
 # Commands for group: org
 
 
+def set_org_name(required):
+    """Generate a callback that enforces the 'required' rule as required"""
+    # could be generalized to work for any mutex pair (or list) but no obvious need
+    def callback(ctx, param, value):
+        """Callback which enforces mutex and 'required' behaviour (if required)."""
+        prev_value = ctx.params.get("org_name")
+        if value and prev_value and prev_value != value:
+            raise click.UsageError(
+                f"Either ORGNAME or --org ORGNAME should be supplied, not both ({value}, {prev_value})"
+            )
+        ctx.params.setdefault("org_name", value)
+        if required and not ctx.params.get("org_name"):
+            raise click.UsageError("Missing argument 'ORGNAME'")
```

Comment:
```suggestion
            raise click.UsageError("Please specify ORGNAME or --org ORGNAME")
```

(Where possible exceptions should suggest what to do, not just say what was wrong.)